src/libraries/System.Private.Xml/src/System/Xml/Core/LocalAppContextSwitches.cs

```diff
@@ -1,60 +1,45 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Runtime.CompilerServices;
-
-namespace System
+namespace System.Xml
 {
-    internal static partial class LocalAppContextSwitches
+    internal static class LocalAppContextSwitches
     {
-        private static int s_dontThrowOnInvalidSurrogatePairs;
-        public static bool DontThrowOnInvalidSurrogatePairs
-        {
-            [MethodImpl(MethodImplOptions.AggressiveInlining)]
-            get
-            {
-                return GetCachedSwitchValue("Switch.System.Xml.DontThrowOnInvalidSurrogatePairs", ref s_dontThrowOnInvalidSurrogatePairs);
-            }
-        }
+        public static bool DontThrowOnInvalidSurrogatePairs { get; } =
+            AppContext.TryGetSwitch(
+                switchName: "Switch.System.Xml.DontThrowOnInvalidSurrogatePairs",
+                isEnabled: out bool value)
+            ? value : false;
+
+        public static bool IgnoreEmptyKeySequences { get; } =
+            AppContext.TryGetSwitch(
+                switchName: "Switch.System.Xml.IgnoreEmptyKeySequences",
+                isEnabled: out bool value)
+            ? value : false;
+
+        public static bool IgnoreKindInUtcTimeSerialization { get; } =
+            AppContext.TryGetSwitch(
+                switchName: "Switch.System.Xml.IgnoreKindInUtcTimeSerialization",
+                isEnabled: out bool value)
+            ? value : false;
 
-        private static int s_ignoreEmptyKeySequences;
-        public static bool IgnoreEmptyKeySequences
-        {
-            [MethodImpl(MethodImplOptions.AggressiveInlining)]
-            get
-            {
-                return GetCachedSwitchValue("Switch.System.Xml.IgnoreEmptyKeySequencess", ref s_ignoreEmptyKeySequences);
```

Comment:
NOTE: there used to be a typo in the switch name here, I have fixed it here according to how it's documented:
https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element